modules/candidate_parameters/ajax/getData.php

```diff
@@ -445,33 +436,39 @@ function getConsentStatusFields()
  * Handles the fetching of Consent Status history
  *
  * @param int   $candID   current candidate's ID
- * @param array $consents consent values
+ * @param array $consentValue consent values
  *
  * @throws DatabaseException
  *
  * @return array
  */
-function getConsentStatusHistory($candID, $consents)
+function getConsentStatusHistory($pscid)
 {
     $db =& \Database::singleton();
 
-    $commentHistory = array();
-
-    foreach ($consents as $consent => $label) {
-        $unformattedComments = $db->pselect(
-            "SELECT entry_staff, data_entry_date, "
-            . $db->escape($consent) . ", "
-            . $db->escape($consent . '_date') . ", "
-            . $db->escape($consent . '_withdrawal')
-            ." FROM consent_info_history WHERE $consent IS NOT NULL and CandID=:cid",
-            array('cid' => $candID)
-        );
-
-        $unformattedComments['label']       = $label;
-        $unformattedComments['consentType'] = $consent;
-
-        array_push($commentHistory, $unformattedComments);
+    $historyData = $db->pselect(
+        "SELECT * 
+         FROM candidate_consent_type_history 
+         WHERE PSCID=:pscid 
+         ORDER BY EntryDate ASC",
+        array('pscid' => $pscid)
+    );
+    
+    $formattedHistory = [];
+    foreach ($historyData as $key => $entry) {
+      $history = [
```

Comment:
array within array within array to map to front-end elements without changing jsx